xontrib/voxapi.py

```diff
@@ -7,16 +16,45 @@
 
 from xonsh.platform import ON_POSIX, ON_WINDOWS, scandir
 
+# This is because builtins aren't globally created during testing.
+# FIXME: Is there a better way?
```

Comment:
No, because voxapi is never loaded as a xontrib, just as a Python module.

The test infrastructure never installs builtins globally, so anything core that's tested can't expect `events` (or `aliases`) to be available.